### ISSUE TITLE
Twist VCO: VCV Rack is crashing when entering modulation assignment mode

### ISSUE THREAD

**Issue by yakimanskiyav** [surge-synthesizer/surge-rack#661]
Hello,

When entering "modulation assignment mode" (blue led on) i have a VCV Rack crashing in 1 second (almost always, but not every time).
Constantly crashing when trying to assign (blue led, clicking on param i want to modulate) a modulation to any param (say, "segment")

```
OS: Manjaro (5.15.78-1)
VCV Rack version: 2.2.0
Surge: 2.1.beta-0-nightly-09e4ace
```

**Comment by baconpaul** [comment]
do you get a stack trace? can you share your log? thank you

**Comment by baconpaul** [comment]
and is it only with twist or do you see this with other modules also?

**Comment by baconpaul** [comment]
oh shit i see it

**Comment by baconpaul** [comment]
This is the fix in #636 / 5bb87cc34e820a8e9cfac14662eaf3a2593e5bc1 not working with custom formatters.

I'll revert that for now and make a 2.0.3

**Comment by baconpaul** [comment]
OK this will be fixed in the next nightly and also in the soon-to-be-released-on-this-website 2.0.3.0 release

thank you. some last minute changes in the library and sdk had us scrambling and this commit was wrong. if you hadn't reported it it would have made it to the first library release. Appreciated.

**Comment by baconpaul** [comment]
OK I'm going to close this issue since 636 is reopened and that crashing hash is backed out and we are lined up to push the crash-free version to the library.
Thanks so much @yakimanskiyav - great timing :)
